lib/collection/src/collection.rs

```diff
@@ -139,34 +137,39 @@ impl Collection {
             });
         }
 
+        // Retrieve 1 extra point to determine proper `next_page_offset`
```

Comment:
Is this an unrelated issue existing in the main branch?

Reply:
no, it's due to the change, that we can not calculate the next ID anymore

Reply:
got it thanks :+1: